### PR TITLE
Fixing remaining LGTM alerts

### PR DESCRIPTION
**Pull request by imsolost**
Hi again! I am fixing the remaining alerts from [LGTM.com](https://lgtm.com/projects/g/vmware/clarity/alerts/?mode=tree), including adding a .lgtm.yml file to filter out certain unnecessary queries for your project. Let me know what you think.

### CODE REVIEW

**Comment by imsolost** [comment]
No worries. If you do decide to use LGTM you should think about using it as a tool for [automatic code review and continuous integration](https://lgtm.com/projects/g/vmware/clarity/ci/) so you can catch errors in pull requests before they are merged. It can be a good tool for catching potential errors or security vulnerabilities in a codebase.

File: .lgtm.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+queries:
```
Comment (gnomeontherun) [inline .lgtm.yml]: Please remove this file, we don't anticipate using this service right now.

Comment (imsolost) [reply .lgtm.yml]: No problem. I've removed this file.

**Comment by gnomeontherun** [comment]
Thanks! Last request to squash please :)

**Comment by imsolost** [comment]
All squashed!

**Comment by gnomeontherun** [comment]
Thanks @imsolost!

**Pull request merged**